### DISCUSSION TITLE
Bitcoin accounts view should show raw amounts

### DISCUSSION THREAD

**Discussion by aspiers** [rotki/rotki#8983] [Ideas]
When expanding individual accounts within the "EVM accounts" view, there is an "Amount" column helpfully showing the amount of the token held:

![image](https://github.com/user-attachments/assets/5f2645a1-0a04-441d-8b5f-2ef2da6fc8c3)

However, there is no such "Amount" column in the "Bitcoin accounts" view; instead there is only a rather unhelpful "Assets" column which is blank, except for in the expanded sub-account view where it always shows a BTC logo and strangely requires the user to mouse over that logo in order to see the true amount in the sub-account:

![image](https://github.com/user-attachments/assets/34fc3fbb-f625-4402-aa5c-1b03a0a36682)

Furthermore, given that Bitcoin doesn't support any tokens/coins other than BTC itself (well, unless Rotki plans to support BRC-20 in the future), it probably makes sense to show the raw amount of BTC not just in the expanded view per account (i.e. the lower arrow in the above screenshot) but also the total for each account (i.e. the upper arrow in the above screenshot).

In fact, the existing "Assets" column could probably be scrapped entirely in favour of the proposed "Amount" column, because Bitcoin is not a multi-asset chain.

**Comment by LefterisJP** [comment]
As I said in Discord. I am of the opinion you are right and **BIG IF** we ever support bitcoin tokens, the UI should change then. Now it should be probably raw bitcoin amount. But wonder how to show it compare to the other EVM table where we do have the various assets. Just the bitcoin icon on the left and instead of next an amount column and next a USD value column?

**Comment by lukicenturi** [comment]
Should be fixed by https://github.com/rotki/rotki/pull/8999
